src/plugins/intel_gpu/tests/functional/remote_blob_tests/gpu_remote_tensor_tests.cpp

```diff
@@ -329,6 +341,280 @@ TEST_P(OVRemoteTensorInputBlob_Test, smoke_canInputRemoteTensor) {
     }
 }
 
+TEST_P(OVRemoteTensorInputBlob_Test, smoke_canInputOutputRemoteTensor) {
+#if defined(ANDROID)
```

Comment:
[random spot]
Actually, it is not connected to this changes, but it looks like we don't have proper handling for RemoteTensors with device memory in case if output primitive has CPU implementation (like DO, NMS etc...), do we?